airflow/providers/discord/notifications/discord.py

```diff
@@ -0,0 +1,81 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from airflow.compat.functools import cached_property
+from airflow.exceptions import AirflowOptionalProviderFeatureException
+
+try:
+    from airflow.notifications.basenotifier import BaseNotifier
+except ImportError:
+    raise AirflowOptionalProviderFeatureException(
+        "Failed to import BaseNotifier. This feature is only available in Airflow versions >= 2.6.0"
+    )
+
+from airflow.providers.discord.hooks.discord_webhook import DiscordWebhookHook
+
+ICON_URL: str = "https://raw.githubusercontent.com/apache/airflow/2.5.0/airflow/www/static/pin_100.png"
```

Comment:
I see this constant being repeated in multiple files (at least in the notifications modules). Is there a common place we could move this to? So that if we want to update this later e.g. change version from 2.5.0 to something else, we would need to update that in only one place.

Reply:
cc: @eladkal @hussein-awala WDYT?

Reply:
Sure @pankajkoti 
Here's the screenshot:

![image](https://github.com/apache/airflow/assets/77084039/b4fede03-0a47-4714-8df2-fcb2492e4421)


I agre with u to place this ICON in a common place, but where

Reply:
@potiuk any recommendations about the icon image? We need to place it somewhere that is considered public API

Reply:
no recommendation. Idelly somewhere wher the target service publishes it as "service" image.

Reply:
@vargacypher 
Lets change the URL to 
`https://raw.githubusercontent.com/apache/airflow/main/airflow/www/static/pin_100.png`
Similar to what we have in Slack provider

https://github.com/apache/airflow/blob/9276310a43d17a9e9e38c2cb83686a15656896b2/airflow/providers/slack/operators/slack.py#L124